src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +44,34 @@
      * Андрей - 600
      * София - 900</p>
      */
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+        final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
move it in the class
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names